### ISSUE TITLE
Mouse related keys are not working.

### ISSUE THREAD

**Issue by Thomas-678** [sylecn/ff-nextpage#48]
The URL of a problematic page: 
All webpages. 

Describe what's wrong on that page: 
Keyboard bindings work fine.
But 

> \<mouse-4\>, \<mouse-5\>, \<wheel-up\>, \<wheel-down\>

 don't work.
An example of my user config: 

> (bind "\<mouse-5\>" 'nextpage)

Tell me your browser version, OS version and nextpage addon version:
Chrome 89.0.4389.114 (64 bit). Win10
Addon version: 2.9.2

**Comment by sylecn** [comment]
I can't reproduce this issue.
Are you using nextpage with other extensions that may have conflict with said hotkeys?

You may add this in the beginning of nextpage user preferences to enable debug, you may see debug output in javascript console. Check whether you can see mouse or wheel events.

```
(enable-debug)
(enable-debug-for-key-events)
(bind "<mouse-5>" 'nextpage)
(bind "<wheel-down>" 'nextpage-maybe)
```

Note that mouse-5 is only available on mouses that have at least 5 buttons (usually 2 side buttons). If you have a common 2 button mouse with wheel, you may want to bind ```<wheel-down>``` instead.

**Comment by Thomas-678** [comment]
I enabled debug and tested that \<mouse-1\>, \<wheel-up\>, \<wheel-down\> are now working.
But \<mouse-2\> to \<mouse-5\> aren't detected in console log. I tested this while disabling all other chrome extensions.
My mouse has two side buttons.

Currently, I use a workaround where I use AutoHotkey to rebind mouse 5 to a key on keyboard.

**Comment by sylecn** [comment]
Thanks for reporting back.

mouse-2 to mouse-5 support should be fixed in nextpage v2.10.1.

mouse-2 still doesn't work in edge browser in windows. Since you use chrome, this issue shouldn't affect you.
